Plugin.cs

```diff
@@ -133,6 +137,36 @@ private void OnCommand(string command, string args)
                     this.configuration.Save();
                     PrintSetting($"{Name} {verboseName}", this.configuration.verbose);
                     return;
+                case whitelistName:
+                    if(argArray.Length == 1){
```

Comment:
Your `if`, `then`, `while`, `for`, etc blocks are all formatted differently than the rest of the file.